src/KeyVault/KeyVault/Commands/Certificate/Policy/SetAzureKeyVaultCertificatePolicy.cs

```diff
@@ -313,5 +314,20 @@ public override void ExecuteCmdlet()
                 }
             }
         }
+
+
+        private void ValidateArguments(int? RenewAtNumberOfDaysBeforeExpiry, int? RenewAtPercentageLifetime)
```

Comment:
```suggestion
        private void ValidateArguments()
```
This function should not be designed for validating RenewAtNumberOfDaysBeforeExpiry and RenewAtPercentageLifetime only even though it's for these two parameters for the time being. We can remove them from argument list as these two parameters are available in class private function.